src/main/java/helper/Constant.java

```diff
@@ -4,15 +4,16 @@
 
     //Variables
     public static final String RAILWAY_URL = "http://www.railway2.somee.com/Page/HomePage.cshtml";
-    public static final String USERNAME = "tnminh1997123@gmail.com";
     public static final String PASSWORD = "minhvip1997";
     public static final String TEST_RESOURCES_PATH = "/src/test/resources/";
+    public static final String USERNAME = "tnminh1997123@gmail.com";
     public static final int TIME_WAITS_DRIVER = 30;
     public static final int TIMES_WAIT_ELEMENTS = 120;
     public static final int LONG_TIME_WAIT = 30;
     public static final String DATE_FORMAT = "M/d/yyyy";
 
     //Message
+    public static final String FAIL_MSG_LOGIN_INVALID_ACCOUNT = "There was a problem with your login and/or errors exist in your form.";
```

Comment:
DO NOT name the variables like this.

INVALID_ACCOUNT_MSG is good enough.

Reply:
Thanks for your review. I just have fixed and pushed into this branch. Please review my newest update.